src/plugins/data_views/server/rest_api_routes/get_data_views.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { UsageCounter } from '@kbn/usage-collection-plugin/server';
+import { schema } from '@kbn/config-schema';
+import { IRouter, StartServicesAccessor } from '@kbn/core/server';
+import { DataViewsService } from '../../common';
+import { handleErrors } from './util/handle_errors';
+import type { DataViewsServerPluginStartDependencies, DataViewsServerPluginStart } from '../types';
+import {
+  SERVICE_PATH,
+  SERVICE_PATH_LEGACY,
+  SERVICE_KEY_MULTIPLE,
+  SERVICE_KEY_MULTIPLE_LEGACY,
+} from '../constants';
+
+interface GetDataViewsArgs {
+  dataViewsService: DataViewsService;
+  usageCollection?: UsageCounter;
+  counterName: string;
+  size: number;
+}
+
+export const getDataViews = async ({
+  dataViewsService,
+  usageCollection,
+  counterName,
+  size,
+}: GetDataViewsArgs) => {
+  usageCollection?.incrementCounter({ counterName });
+  return dataViewsService.find('', size);
```

Comment:
lets do `dataViewsService.getIdsWithTitle()` - I now see how my terse issue was incomplete. This is more about discovering which data views exist rather than grabbing multiple data views. Grabbing a bunch of data views could also be stressful on the cluster due to field_caps queries.